src/main/java/com/google/api/codegen/discogapic/transformer/DiscoGapicNamer.java

```diff
@@ -32,11 +32,46 @@ public String getFieldVarName(String fieldName) {
 
   /** Returns the resource getter method name for a resource field. */
   public String getResourceGetterName(String fieldName) {
-    return publicMethodName(Name.anyCamel("get", fieldName));
+    Name name;
+    if (fieldName.contains("_")) {
+      name = Name.anyCamel(fieldName.split("_"));
```

Comment:
Some parameter names defined in the Discovery doc have an underscore, ex "order_by." All the Name.*Camel methods will throw an exception if the input has underscores (because it's not in camelcase).